### PR TITLE
Fix Issue #1082 added lang attribute for language specific pronunciation

### PR DESCRIPTION
**Pull request by thattonBL**
- lang attribute added to parent div of all UV elements which should cascade down

### CODE REVIEW

**Comment by demiankatz** [comment]
Thanks, @thattonBL, this seems to be working as expected (in terms of adding the lang attribute -- the interaction of this attribute with various screen readers is beyond my ability to test). Before we merge this, though, I wonder if we should add better lang attribute support to iiif-metadata-component, since the metadata in that panel may be a different language from the interface, and those elements do not currently appear to be labeled. I think we have enough data in the panel to label each value, so we should probably start doing that. I don't have time to investigate it just yet, but it may not be too difficult when time permits!

**Comment by LanieOkorodudu** [comment]
@thattonBL and @demiankatz, I've tested this fix on the Vercel app, verifying the fix in Dev Tools and comparing  it with the dev branch. All changes function as expected. I validated the lang attribute across all available languages, confirming it is correctly applied and behaving as intended.

**Comment by demiankatz** [comment]
So I think the question now is whether we merge this as-is or wait until some more nuance has been added. Do you have an opinion, @scoutb-cogapp?

**Comment by demiankatz** [comment]
When #1237 is merged, the metadata component should put appropriate `lang` attributes on metadata. That, in combination with this PR, will hopefully resolve the whole issue... though I'm not in a good position to test it myself!

File: src/content-handlers/iiif/modules/uv-shared-module/BaseExtension.ts
Comment (demiankatz) [inline src/content-handlers/iiif/modules/uv-shared-module/BaseExtension.ts:115]: Minor style suggestion:

```suggestion
    } else {
```

**Comment by demiankatz** [comment]
I've merged the dev branch into this PR so that the metadata component fixes are incorporated. @LanieOkorodudu has offered to do another round of testing.

**Review by LanieOkorodudu — APPROVED** [review]
@demiankatz, I tested the lang attribute again using the developer tools and confirmed that it updates correctly when the language is changed in the settings. I also checked with a screen reader to verify if it reads the content in the selected language, and it does. The only observation is a slight inconsistency in the accent, it reads the content accurately, but the accent may not always sound fully natural. Otherwise, the functionality performs as expected, with the language updating correctly each time.

**Pull request merged**
